### ISSUE TITLE
Dragging window to another display makes Icon show up incorrectly

### ISSUE THREAD

**Issue by denis-ismailaj** [JetBrains/compose-jb#677]
When I drag the window from my main display (2560x1600) to my secondary one (2560 x 1440), the `Icon` composable is duplicated and is shown at another size. It looks fine when I drag it back to the first screen though.

![Image](https://user-images.githubusercontent.com/20902736/118873085-9ca6cd00-b8e9-11eb-832c-12a22df6ffba.png)

I'm on Compose version `0.4.0-build190`.

MInimal repro code:
```
fun main() = Window {
    Icon(
        imageVector = Icons.Default.Refresh,
        contentDescription = null
    )
}
```

**Comment by igordmn** [comment]
Fixed in 0.4.0-build212

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-5411">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
